community.md

```diff
@@ -0,0 +1,143 @@
+            <img src="../assets/images/community/youtube.png" />
+          </a>
+        </div>
+      </div>
+    </div>
+    <div class="community-content-section">
+      <h2>Contribute</h2>
+      <div class="community-content-container contribute-content-container">
+        <div class="contribute-text-container">
+          <div class="contribute-github-button d-flex flex-row">
+            <a class="github-button" href="https://github.com/trinodb" data-size="large" aria-label="Follow @trinodb on GitHub">Follow @trinodb</a>
+            <a class="github-button" href="https://github.com/trinodb/trino" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star trinodb/trino on GitHub">Star</a>
+          </div>
+          <p>Help build the next generation of fast distributed SQL query engines for big data analytics. Trino is maintained by hundreds of around the globe.</p>
+          <p>Over the years, the Trino project has developed an extensive vision of the software and a strong development philosophy.</p>
+          <p>Understanding these helps you get up to speed on the software and how the project works quickly.</p>
+          <p><a href="../development/vision.html">Read more about the project vision and development philosophy...</a></p>
+        </div>
+        <div class="d-flex flex-column" style="flex: 1 1 50%;">
+          <div class="contribute-thirds-container">
+            <div class="circle-icon-container circle-bugs">
+              <i class="fa fa-bug fa-lg"></i>
+            </div>
+            <div>
+              <h3>Find and report issues</h3>
+              <p>Report bugs on Github and report security vulnerabilities to <a href="mailto:security@trino.io">security@trino.io</a></p>
+              <a class="github-button" href="https://github.com/trinodb/trino/issues" data-icon="octicon-issue-opened" data-size="large" data-show-count="true" aria-label="Issue trinodb/trino on GitHub">Issue</a>
+            </div>
+          </div>
+          <div class="contribute-thirds-container">
+            <div class="circle-icon-container circle-docs">
+              <i class="fa fa-pencil-alt fa-lg"></i>
+            </div>
+            <div>
+              <h3>Write documentation</h3>
+              <p>Contributions are not just about code. Help the community understand how to use our distributed SQL engines by adding to <a href="../docs/current/">our docs</a>.</p>
+            </div>
+          </div>
+          <div class="contribute-thirds-container">
+            <div class="circle-icon-container circle-pr">
+              <i class="fa fa-code-branch fa-lg"></i>
+            </div>
+            <div>
+              <h3>Open a pull request</h3>
+              <p>Improve exisiting functionality or even add your own feature.</p>
+              <a href="../development/process.html">Read about our contribution process to learn more...</a>
```

Comment:
Read about our <a href="../development/process.html">contribution process</a>  to learn more.